fog/api/proto/ledger.proto

```diff
@@ -133,7 +133,7 @@ message KeyImageResult {
     /// never be found with the current configuration of the Key Image Service's watcher.
     /// This is fixed32 to avoid leaking information about found / not found in the size of the encrypted
     /// payload.
-    /// The possible values are described in enum TimestampResultCode.
+    /// The possible values are described in enum TimestampResultCode. None of the other codes are going to be used except the key image spent
```

Comment:
this comment isn't right -- KeyImageSpent is a KeyImageResultCode, not a TimestampResultCode

Here's what I suggest:

```
/// This is a legacy result code which was forwarded by ledger server from the Watcher db API if a timestamp is not available.
/// The ledger server now handles all of these errors and the result will always be `TimestampFound`.
/// Clients should ignore this value, and in a future revision we may make it always zero.
```

Reply:
We should also update the comment for `timestamp`:

```
    /// The timestamp of the block containing this key image.
    /// The value is u64::MAX if the timestamp cannot be found.
    /// If the timestamp cannot be found, that represents an internal error of the server
    /// which should be reported to the developers.
    /// Note: The timestamps are based on untrusted reporting of time from the consensus validators.
    /// Represented as seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z.
```

Reply:
made the change and pushing to branch